apps/system/components/input_method.css

```diff
@@ -12,7 +12,7 @@ input-method {
 }
 
 input-method:not(.no-animation) {
-  transition: transform 0.15s;
+  transition: transform 0.3s cubic-bezier(0.2, 0.9, 0.1, 1);
```

Comment:
unrelated change, please revert in this PR

Reply:
oh i forgot about that